patches/minecraft/net/minecraft/tags/ITag.java.patch

```diff
@@ -1,10 +1,15 @@
 --- a/net/minecraft/tags/ITag.java
 +++ b/net/minecraft/tags/ITag.java
-@@ -46,6 +_,7 @@
+@@ -44,8 +_,11 @@
+       return Tag.func_241286_a_(p_232946_0_);
+    }
  
-    public static class Builder {
+-   public static class Builder {
++   public static class Builder implements net.minecraftforge.common.extensions.IForgeRawTagBuilder {
        private final List<ITag.Proxy> field_232953_a_ = Lists.newArrayList();
 +      private boolean replace = false;
++      private final List<ITag.Proxy> removeEntries = Lists.newArrayList();
++      public List<ITag.Proxy> getRemoveEntries() { return this.removeEntries; } // forge -- track additions to the "remove" list during datageneration
```

Comment:
This should be a read-only view, see [this](https://github.com/MinecraftForge/MinecraftForge/pull/7911/files#diff-f8049b7220c9c3128f74e20f2128209a7a56cc09f4306035144cb8ed4dfbfdb7R9-R15).